### ISSUE TITLE
Bug: NPM EINTEGRITY during package install

### ISSUE THREAD

**Issue by sandermarechal** [FortAwesome/Font-Awesome#19194]
### Bug description

We have a FontAwesome PRO plan. I am trying to install FontAwesome Pro using NPM. It works fine on our dev machines, but
not in our CI/CD environment (gitlab). I get the following error:

```
  npm ERR! code EINTEGRITY
  npm ERR! sha512-ApOqpDdKgA79xfLZH3B5PucZxj+TZyQUSrZ4bKkbJCR+zjmveQ4/gp/uXc5
  bNNhsdtJUy16BYJ/lAVydca2Y5Q== integrity checksum failed when using sha512:
  wanted sha512-ApOqpDdKgA79xfLZH3B5PucZxj+TZyQUSrZ4bKkbJCR+zjmveQ4/gp/uXc5bN
  NhsdtJUy16BYJ/lAVydca2Y5Q== but got sha512-z4PhNX7vuL3xVChQ1m2AB9Yg5AULVxXc
  g/SpIdNs6c5H0NE8XYXysP+DGNKHfuwvY7kxvUdBeoGlODJ6+SfaPg==. (0 bytes)
```

The hash `sha512-ApO...` is the hash of the fontawesome-pro 5.15.4 package that
I installed in my dev environment. I have no idea where the `sha512-z4P...` hash
is coming from.

NPM is configured correctly on my CI/CD according to:
https://fontawesome.com/v5/docs/web/setup/use-package-managers

On CI/CD the config of my .npmrc is:

  cache=/root/npm-cache
  @fortawesome:registry=https://npm.fontawesome.com/
  //npm.fontawesome.com/:_authToken=[MASKED]

Can you help? Do you know where the `sha512-z4P...` hash is coming from?

### Font Awesome version

5.15.4

### Serving

Self-hosted

### Implementation

CSS

### Browser and Operating System

- Node.js 10 + NPM 6

### Web bug report checklist

- [ ] I have included a test case because my odds go _way_ up that the team can fix this when I do
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by sandermarechal** [comment]
Also, I have tried contacting PRO support at the hello@ e-mail address multiple times over the last few weeks, but I have never received any reply or acknowledgement. Not even an automated confirmation.

**Comment by tagliala** [comment]
Thanks for the heads-up, will escalate

**Comment by robmadole** [comment]
@sandermarechal we'll have to get sensitive information from you in order to continue. Stuff that you don't want to post in a public Github issue.

So let's figure out why we aren't getting your emails into Helpscout. Can you send me the email address that you used to email `hello@fontawesome.com` to rob at fontawesome.com?

**Comment by sandermarechal** [comment]
Yes. I sent the e-mails from sander (at) prezent.nl. Our pro support account is registered at projectmanagement (at) prezent.nl

**Comment by sandermarechal** [comment]
I previously sent e-mails on 29-06-2022 at 10:46 CET and 13-07-2022 at 11:36 CET

**Comment by sandermarechal** [comment]
@robmadole I sent you an email on 20 july at 17:49 CEST. Did you recieve it? Or is there a problem sending mail from us to all fontawesome.com addresses? Below is the delivery line from out company maillogs (I munged your e-mail address):

```
Jul 20 17:49:30 smtp4 postfix/smtp[19994]: 0CB951AB: to=<[rob [at] fontawesome.com](mailto:rob [at] fontawesome.com)>, relay=aspmx.l.google.com[142.250.102.27]:25, delay=0.76, delays=0.08/0.02/0.16/0.5, dsn=2.0.0, status=sent (250 2.0.0 OK  1658332170 c24-20020a170906341800b0072b4cba9671si20375203ejb.186 - gsmtp)
```

**Comment by robmadole** [comment]
@sandermarechal the email was flagged as Spam by Google. I see it now. I just checked Helpscout for emails from you and none are making it through.

I've marked message from you as "Not Spam". So I'll follow up with a reply to your email from there.

**Comment by sandermarechal** [comment]
This appears to be a bug specific to the node 10 and npm 6 versions shipped with Ubuntu 20.04 focal

**Comment by tagliala** [comment]
> This appears to be a bug specific to the node 10 and npm 6 versions shipped with Ubuntu 20.04 focal

Thanks, closing here for the moment
